EnglishTranslation-master/SKILL.tsv

```diff
@@ -4582,7 +4582,7 @@ SKILL_20150714_004581	Increases the damage dealt on an enemy with [Barbed Arrow]
 SKILL_20150714_004582	Increases the damage dealt on an enemy with [Crossfire] by 1% per attribute level.
 SKILL_20150714_004583	Increases the damage dealt on an enemy with [Magic Arrow] by 1% per attribute level.
 SKILL_20150714_004584	Increases the damage dealt on an enemy with [Divine Machine Arrow] by 1% per attribute level.
-SKILL_20150714_004585	Increases the time duration of the [Bleeding] status when an enemy is hit by [Broad Head] by 2 seconds per attribute level.
+SKILL_20150714_004585	Increases the duration of [Bleeding] when an enemy is hit by [Broadhead] by 2 seconds per attribute level.
```

Comment:
Increases the [Bleeding] duration of [Broadhead] by 2 seconds per attribute level. ?

That way it doesn't seem like the act of hitting with Broadhead extends the current bleeding stack.